packages/clay-toolbar/src/Label.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayLabel from '@clayui/label';
+import classNames from 'classnames';
+import React from 'react';
+
+export interface IProps extends React.ComponentProps<typeof ClayLabel> {
+	/**
+	 * FLag that determines if the Link will have a `component-label` class, allowing CSS overrides.
+	 */
+	componentLabel?: boolean;
+}
+
+const Label: React.FunctionComponent<IProps> = ({
+	children,
+	className,
+	componentLabel,
+	...otherProps
+}) => (
+	<ClayLabel
+		className={classNames(className, 'tbar-label', {
+			'component-label': componentLabel,
```

Comment:
I think we can just always leave this class as active, seems like itll always be there for labels in a tbar. This means we can remove the prop as well.